### PR TITLE
(spanish) 8-Reinforcement/2-Gym/assignment.md

### PR DESCRIPTION
**Pull request by hereje**
Translate file `8-Reinforcement/2-Gym/assignment.md` to spanish

### CODE REVIEW

File: 8-Reinforcement/2-Gym/translations/assignment.es.md [hunk 1]
```diff
@@ -1,43 +1,44 @@
...
 |---|---|
-| 0 | Accelerate to the left |
-| 1 | Do not accelerate |
-| 2 | Accelerate to the right |
+| 0 | Acelerar a la izquierda |
+| 1 | No acelerar |
+| 2 | Acelerar a la derecha |
 
-The main trick of this problem is, however, that the car's engine is not strong enough to scale the mountain in a single pass. Therefore, the only way to succeed is to drive back and forth to build up momentum.
+El truco principal de este problema es, no obstante, que el motor del carrito no es lo suficientemente potente para escalar la montaña en una sola pasada. Por lo tanto, la única forma de lograrlo es conducir de hacia atrás y  adelante para generar impulso.
```
Comment (Stevengez) [inline 8-Reinforcement/2-Gym/translations/assignment.es.md:19]:
```suggestion
El truco principal de este problema es, no obstante, que el motor del carrito no es lo suficientemente potente para escalar la montaña en una sola pasada. Por lo tanto, la única forma de lograrlo es conducir hacia atrás y adelante para generar impulso.
```

File: 8-Reinforcement/2-Gym/translations/assignment.es.md [hunk 2]
```diff
@@ -1,43 +1,44 @@
...
-|-----|--------------|-----|-----|
-|  0  | Car Position | -1.2| 0.6 |
-|  1  | Car Velocity | -0.07 | 0.07 |
+| Num | Observación           | Min   | Max  |
+|-----|-----------------------|-------|------|
+|  0  | Posición del carrito   | -1.2  | 0.6  |
+|  1  | Velocidad del carrito | -0.07 | 0.07 |
 
-Reward system for the mountain car is rather tricky:
+El sistema de recompensas para el carrito de montaña es bastante difícil:
```
Comment (Stevengez) [inline 8-Reinforcement/2-Gym/translations/assignment.es.md:28]:
```suggestion
El sistema de recompensas para el carrito de montaña es engañoso:
```

File: 8-Reinforcement/2-Gym/translations/assignment.es.md [hunk 3]
```diff
@@ -1,43 +1,44 @@
...
 
- * Reward of 0 is awarded if the agent reached the flag (position = 0.5) on top of the mountain.
- * Reward of -1 is awarded if the position of the agent is less than 0.5.
+* La recompensa de 0 es otorgada si el agente alcanzó la bandera (position = 0.5) en la cima de la montaña.
```
Comment (Stevengez) [inline 8-Reinforcement/2-Gym/translations/assignment.es.md:30]:
```suggestion
* La recompensa de 0 es otorgada si el agente alcanzó la bandera (posición = 0.5) en la cima de la montaña.
```

File: 8-Reinforcement/2-Gym/translations/assignment.es.md [hunk 4]
```diff
@@ -1,43 +1,44 @@
...
 
-Adapt our reinforcement learning algorithm to solve the mountain car problem. Start with existing [notebook.ipynb](notebook.ipynb) code, substitute new environment, change state discretization functions, and try to make existing algorithm to train with minimal code modifications. Optimize the result by adjusting hyperparameters.
+## Instrucciones
 
-> **Note**: Hyperparameters adjustment is likely to be needed to make algorithm converge. 
-## Rubric
+Adapta nuestro algoritmo de aprendizaje reforzado para resolver el problema del carrito de montaña. Comienza con el código existente en [notebook.ipynb](../notebook.ipynb), substituye el nuevo ambiente, cambia las funciones de discretización de estado, e intenta hacer que el algoritmo existente entrene con mínimas modificaciones al código. Optimiza el resultado al ajustar los hiperparámetros.
 
-| Criteria | Exemplary | Adequate | Needs Improvement |
+> **Nota**: El ajuste de los hiperparámetros es probable sea necesario para hacer que el algoritmo converja.
```
Comment (Stevengez) [inline 8-Reinforcement/2-Gym/translations/assignment.es.md:39]:
```suggestion
> **Nota**: Es probable que ajustar los hiperparámetros sea necesario para hacer que el algoritmo converja.
```

File: 8-Reinforcement/2-Gym/translations/assignment.es.md [hunk 5]
```diff
@@ -1,43 +1,44 @@
...
 | -------- | --------- | -------- | ----------------- |
-|          | Q-Learning algorithm is successfully adapted from CartPole example, with minimal code modifications, which is able to solve the problem of capturing the flag under 200 steps. | A new Q-Learning algorithm has been adopted from the Internet, but is well-documented; or existing algorithm adopted, but does not reach desired results | Student was not able to successfully adopt any algorithm, but has mede substantial steps towards solution (implemented state discretization, Q-Table data structure, etc.) |
+|          | El algoritmo Q-Learning se adaptó de forma exitosa a partir del ejemplo CartPole, con mínimas modificaciones al código, el cual es capaz de resolver el problema al capturar la bandera con menos de 200 pasos. | Se adoptó un nuevo algoritmo Q-Learning de internet, pero está bien documentado; o se adoptó el algoritmo existente, pero no cumple los resultados deseados | El estudiante no fue capaz de adoptar algún algoritmo de forma exitosa, pero ha hecho pasos substanciales hacia la solución (implementó la discretización de estado, la estructura de datos de Q-Table, etc.) |
```
Comment (Stevengez) [inline 8-Reinforcement/2-Gym/translations/assignment.es.md:44]:
```suggestion
|          | El algoritmo Q-Learning se adaptó de forma exitosa a partir del ejemplo CartPole, con mínimas modificaciones al código, el cual es capaz de resolver el problema al capturar la bandera con menos de 200 pasos. | Se adoptó un nuevo algoritmo Q-Learning de internet, pero está bien documentado; o se adoptó el algoritmo existente, pero no cumple los resultados deseados | El estudiante no fue capaz de adoptar ningun algoritmo de forma exitosa, pero ha hecho pasos substanciales hacia la solución (implementó la discretización de estado, la estructura de datos de Q-Table, etc.) |
```

**Review by Stevengez — CHANGES_REQUESTED** [review]
Great job with the translation!, I added some suggestions to make it even better

**Comment by jlooper** [comment]
thank you for the review!

**Pull request merged**